analysis/trajectory.Rmd

```diff
@@ -10,8 +10,77 @@ bibliography: ../ref.bib
 source("workflows/knitr_options.R")
 ```
 
-*WIP*
+In this chapter, we will walk through a minimal analysis to perform lineage reconstruction and psuedotime inference for single-cells along a continuum. 
+We will demonstrate this with the [`slinghot`](https://bioconductor.org/packages/slingshot) Bioconductor package. 
+The goal of slingshot is to use clusters of cells to uncover global structure and convert this structure into smooth lineages represented by one-dimensional variables, called "pseudotime".
+The package offers tools to learn the cluster relationships in an unsupervised or semi-supervised manner and constructing smooth curves representing each lineage, along with visualization methods for each step.
+Slingshot was designed to model developmental trajectories in scRNA-seq data and serve as a component in an analysis pipeline after dimensionality reduction and clustering. 
 
+We will use the 10X PBMC data in the Bioconductor data package `TENxPBMCData` that we have been using in previous chapters. 
```

Comment:
Use `r Biocpkg("TENxPBMCData")` for package references. Same for slingshot above.

Reply:
fixed